applications/StructuralMechanicsApplication/tests/cpp_tests/test_truss.cpp

```diff
@@ -15,6 +15,72 @@
 #include "structural_mechanics_application_variables.h"
 
 #include "custom_elements/truss_element_3D2N.hpp"
+#include "custom_elements/truss_element_linear_3D2N.hpp"
+
+#include <utility>
+
+namespace
+{
+
+using namespace Kratos;
+
+class StubBilinearLaw : public ConstitutiveLaw
```

Comment:
what does stub mean?

Reply:
In this context, it is meant as a mock object. The intent here is to have a tailored constitutive law that mimics a bilinear law. It only implements the operations that are being used by the unit tests, in the simplest way possible. I'm open to any suggestions for a better name. Thank you.

Reply:
ah ok

how about `MockBilinearLaw` then? :smile: